example/javascriptlib/testing/1-test-suite/foo/src/calculator.ts

```diff
@@ -0,0 +1,20 @@
+export class Calculator {
+    add(a: number, b: number): number {
+        return a + b;
+    }
+
+    subtract(a: number, b: number): number {
+        return a - b;
+    }
+
+    multiply(a: number, b: number): number {
+        return a * b;
+    }
+
+    divide(a: number, b: number): number {
+        if (b === 0) {
+            throw new Error("Division by zero is not allowed");
+        }
+        return a / b;
+    }
```

Comment:
Let's trim this down, just `add` and `divide` (and associated tests) should be enough to demonstrate the build setup

Reply:
Same for the one above